.github/workflows/test.yml

```diff
@@ -0,0 +1,35 @@
+name: Tests
+
+on:
+  # Trigger analysis when pushing in main or pull requests, and when creating
+  # a pull request.
+  push:
+    branches:
+      - main
+  pull_request:
+      types: [opened, synchronize, reopened]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    # Tests for Python
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Python info
+      shell: bash -l {0}
+      run: |
+        which python3
+        python3 --version
+    - name: Upgrade pip and install dependencies
+      run: |
+        python3 -m pip install --upgrade pip setuptools
+        python3 -m pip install -r requirements.txt
+        python3 -m pip install -r requirements-dev.txt
+    - name: Run unit tests
+      run: |
+        export PYTHONPATH=$PYTHONPATH:$PWD
```

Comment:
is this `export` necessary? my local test does not need it

Reply:
I tried, and it is indeed necessary :) See https://github.com/PerfectFit-project/virtual-coach-db/runs/4702749002?check_suite_focus=true

Reply:
then it's necessary :-)